setup.cfg

```diff
@@ -25,6 +25,9 @@ install_requires =
     numpy >=1.9.3, !=*rc*
     matplotlib >=3.1.1, !=3.3.0rc1
     PyYAML >=5.1, !=*rc*
+    scipy
```

Comment:
`scipy` could be removed (see other comments), but could you move pybids and pandas to an extra option (something like "synch" or a better name)?
In this way, they will be installed only if the function will be used.

(unless you feel like rewriting the pandas-based lines to avoid using pandas :stuck_out_tongue_winking_eye: )

Btw, remember to add the extra option in `test` and `all`!

Reply:
Done!